awsprofile/__init__.py

```diff
@@ -78,9 +80,11 @@ def main():
         else:
             os.putenv('AWS_SESSION_TOKEN', creds.token)
 
-    my_env = os.environ.copy()
-    command_status = os.system(command)
-    exit(os.WEXITSTATUS(command_status))
+    try:
+        subprocess.run(command,
```

Comment:
You're absolutely right. `subprocess.call` is more appropriate anyway and supported back to 2.7 (and also 2.6 I think but not 100% sure on that).